spec/controllers/owned_tag_sets_controller_spec.rb

```diff
@@ -81,8 +81,8 @@
         it "then sets the correct tags with the type fandom" do
```

Comment:
These tests started failing after I added mine because they were comparing a list of `tag_set_id` with a list of `id`. These happened to be the same as long as no other tests were creating tag sets. When mine were added, this pushed the `id` and `tag_set_ids` out of sync and revealed the bug.